src/output-json.c

```diff
@@ -1033,7 +1025,13 @@ void JsonFiveTuple(const Packet *p, enum OutputJsonLogDirection dir, json_t *js)
             break;
     }
 
-    json_object_set_new(js, "proto", json_string(proto));
+    if (SCProtoNameValid(IP_GET_IPPROTO(p))) {
+        json_object_set_new(js, "proto", json_string(known_proto[IP_GET_IPPROTO(p)]));
+    } else {
+        char proto[4];
+        snprintf(proto, sizeof(proto), "%"PRIu8"", IP_GET_IPPROTO(p));
```

Comment:
@jlucovsky I now get a warning from my clang
```
  CC       output-json.o
output-json.c:1032:52: warning: format specifies type 'unsigned char' but the argument has type 'int' [-Wformat]
        snprintf(proto, sizeof(proto), "%"PRIu8"", IP_GET_IPPROTO(p));
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~
```

Should we use `PRIu32` instead ?